### PR TITLE
add appendOutputTo method

### PR DESCRIPTION
**Pull request by zhushengwen**
laravel has appendOutputTo is very usefull,I try add the method,please accept.

### CODE REVIEW

File: Event.php [hunk 1]
```diff
@@ -73,12 +79,6 @@ class Event extends Component
      * @var string
      */
     protected $_description;
-    /**
-     * The mutex implementation.
-     *
-     * @var \yii\mutex\Mutex
-     */
-    protected $_mutex;
```
Comment (omnilight) [inline Event.php]: Why did you remove mutex? it is used for withoutOverlapping

File: Event.php [hunk 2]
```diff
@@ -61,6 +61,12 @@ class Event extends Component
      * @var string
      */
     protected $_output = null;
+    /**
+     * The location that output should be sent to.
+     *
+     * @var string
+     */
+    protected $_redirect = '>';
```
Comment (omnilight) [inline Event.php]: Do not see, if it is used in the run method

File: Event.php (hunk 1 above)
Comment (zhushengwen) [reply Event.php]: I am sorry the code should not be deleted.

Comment (zhushengwen) [reply Event.php]: Thanks,I will start a new pull request soon.

**Comment by gietos** [comment]
Closing as duplicate of #37
